src/twyn/file_handler/file_handler.py

```diff
@@ -0,0 +1,42 @@
+import logging
+import os
+from pathlib import Path
+from typing import Protocol
+
+from twyn.base.exceptions import TwynError
+from twyn.file_handler.exceptions import PathIsNotFileError, PathNotFoundError
+
+logger = logging.getLogger()
+
+
+class BaseFileHandler(Protocol):
+    def __init__(self, file_path: str) -> None: ...
+    def read(self) -> str: ...
+    def file_exists(self) -> bool: ...
+
+
+class FileHandlerPathlib(BaseFileHandler):
+    def __init__(self, file_path: str) -> None:
+        self.file_path = Path(os.path.abspath(os.path.join(os.getcwd(), file_path)))
+
+    def read(self) -> str:
+        self._raise_for_file_exists()
+
+        content = self.file_path.read_text()
+        logger.debug("Successfully read content from local dependencies file")
+
+        return content
+
+    def file_exists(self) -> bool:
+        try:
+            self._raise_for_file_exists()
```

Comment:
If we are only using `_raise_for_file_exists` here, why not just simply:

```python
def file_exists(self) -> bool:
   return self.file_path.exists() and self.file_path.is_file()
```

Reply:
It's so it can be also used in `def read()`, so we avoid code repetition.

Reply:
Fair!